### ISSUE TITLE
compose.py failure: module 'build' has no attribute 'get_container_versions'

### ISSUE THREAD

**Issue by jayvdb** [triton-inference-server/server#4263]
**Description**

```py
Traceback (most recent call last):
  File "/path/to/triton-server-repo/compose.py", line 388, in <module>
    get_container_version_if_not_specified()
  File "/path/to/triton-server-repo/compose.py", line 162, in get_container_version_if_not_specified
    _, FLAGS.container_version = build.get_container_versions(
AttributeError: module 'build' has no attribute 'get_container_versions'
```

`get_container_versions` was removed in de061a4 (PR #4174) , but it is still used by `compose.py` on branch `main`, c.f. https://github.com/triton-inference-server/server/blob/db3ca6a/compose.py#L162

**To Reproduce**
Run `python3 compose.py --dry-run` without any other arguments.

**Expected behavior**
No exceptions for use without any args.  If this mode is no longer supported, a nice error message would be great.

**Comment by jayvdb** [comment]
ping @deadeyegoodwin

**Comment by Tabrizian** [comment]
@jbkyang-nvi Can you please take a look at this issue?

**Comment by deadeyegoodwin** [comment]
This should be fixed by #4276, you can try the edit locally to get unblocked.
